tests/justice_board.test.ts

```diff
@@ -0,0 +1,88 @@
+import lodash from 'lodash';
+import request from 'supertest';
+import { Express } from 'express';
+import { Collection } from 'mongodb';
+
+import { getServer } from '../index';
+import { Duty } from '../schemas/duty.zschema';
+import { Soldier } from '../schemas/soldier.zschema';
+import { createTestDutyWithId } from './data/duty.data';
+import { createTestSoldier } from './data/soldier.data';
+import { getSoldiersCollection, getDutiesCollection } from '../db_connection';
+
+let server: Readonly<Express>;
+let soldiersCollection: Readonly<Collection<Soldier>>;
+let dutiesCollection: Readonly<Collection<Duty>>;
+
+beforeAll(async () => {
+  soldiersCollection = await getSoldiersCollection();
+  dutiesCollection = await getDutiesCollection();
+  server = getServer();
+});
+
+beforeEach(async () => {
+  soldiersCollection.deleteMany({});
+  dutiesCollection.deleteMany({});
+});
+
+afterEach(async () => {
+  soldiersCollection.deleteMany({});
+  dutiesCollection.deleteMany({});
+});
+
+describe('GET /justice-board', () => {
```

Comment:
You need to test the service and the router, as the service is about to get a lot thicker </br>